compiler/x/amd64/codegen/TreeEvaluatorTable.hpp

```diff
@@ -35,17 +35,31 @@
    TR::TreeEvaluator::iloadEvaluator,                   // TR::iload
    TR::TreeEvaluator::floadEvaluator,                   // TR::fload
    TR::TreeEvaluator::dloadEvaluator,                   // TR::dload
-   TR::TreeEvaluator::aloadEvaluator,                 // TR::aload
+   TR::TreeEvaluator::aloadEvaluator,                   // TR::aload
    TR::TreeEvaluator::bloadEvaluator,                   // TR::bload
    TR::TreeEvaluator::sloadEvaluator,                   // TR::sload
-   TR::TreeEvaluator::lloadEvaluator,                 // TR::lload
+   TR::TreeEvaluator::lloadEvaluator,                   // TR::lload
+   TR::TreeEvaluator::iReadBarrierEvaluator,            // TR::irdbar
```

Comment:
This is a little unorthodox naming scheme. I believe the convention is to name the evaluators `<ilOpcode>Evaluator` across all the codegens so it is easy to grep around and search for the implementations. This seems to deviate from the naming scheme of all the other IL evaluators (see above and below these ones for example).

Reply:
thanks for the suggestion! modified accordingly.